setup.py

```diff
@@ -9,4 +9,5 @@
       author_email='no-idea@no-server.no-suffix',
       url='https://github.com/rust-lang-nursery/highfive',
       packages=['highfive'],
+      install_requires=['retry'],
```

Comment:
I feel like this project has aimed to avoid adding external dependencies in the past, but that may be in my head. Pinging @nrc, if he has any feelings about this.